source/MaterialXGraphEditor/Graph.cpp

```diff
@@ -3052,8 +3283,156 @@ void Graph::graphButtons()
     ImGui::EndChild();
     ImGui::SameLine(0.0f, 12.0f);
 
-    handleRenderViewInputs(windowPos, screenSize[0], screenSize[1]);
+    if (!isDialogOpen())
+    {
+        handleRenderViewInputs(windowPos, screenSize[0], screenSize[1]);
+    }
+}
+
+void Graph::addCreateDefinitionPopup()
```

Comment:
This is the main UI.
- Version and namespace are optionally enabled UI
- "Custom" library location enabling allows for a file dialog to be used to specific a specific custom location (vs the default user location).